themes/default/content/docs/guides/pulumi-packages/schema.md

```diff
@@ -0,0 +1,321 @@
+| `packageDescription`                        | `string`      | No       | Description for the NPM package.                              |
+| `readme`                                    | `string`      | No       | Readme contains the text for the package's README.md files.   |
+| `dependencies`                              | `map[string]` | No       | NPM dependencies to add to package.json.                      |
+| `devDependencies`                           | `map[string]` | No       | NPM dev-dependencies to add to package.json.                  |
+| `peerDependencies`                          | `map[string]` | No       | NPM peer-dependencies to add to package.json.                 |
+| `resolutions`                               | `map[string]` | No       | NPM resolutions to add to package.json                        |
+| `typescriptVersion`                         | `string`      | No       | A specific version of TypeScript to include in package.json.  |
+| `moduleToPackage`                           | `map[string]` | No       | A map containing overrides for module names to package names. |
+| `compatibility`                             | `string`      | No       | Toggle compatibility mode for a specified target.             |
+| `disableUnionOutputTypes`                   | `boolean`     | No       | Disable support for unions in output types.                   |
+| `containsEnums`                             | `boolean`     | No       | An indicator for whether the package contains enums.          |
+
+For `python`:
+
+| Property               | Type          | Required | Description                                                                                                              |
+|------------------------|---------------|----------|--------------------------------------------------------------------------------------------------------------------------|
+| `packageName`          | `string`      | No       | PackageName is an override for the name of the generated python package.                                                 |
+| `requires`             | `map[string]` | No       | Description for the NPM package.                                                                                         |
+| `readme`               | `string`      | No       | Readme contains the text for the package's README.md files.                                                              |
+| `moduleNameOverrides`  | `map[string]` | No       | Optional overrides for Pulumi module names (e.g. `{ "flowcontrol.apiserver.k8s.io/v1alpha1": "flowcontrol/v1alpha1" }`). |
+| `compatibility`        | `string`      | No       | Toggle compatibility mode for a specified target.                                                                        |
+| `usesIOClasses`        | `boolean`     | No       | Deprecated: This bool is no longer needed since all providers now use input/output classes.                              |
```

Comment:
I don't think we need to document this. This is deadcode that needs to be deleted from the implementation.